posix.mak

```diff
@@ -250,11 +250,13 @@ clean:
 	rm -f docs.json docs-prerelease.json dpl-docs/dpl-docs
 	@echo You should issue manually: rm -rf ${DMD_DIR}-${LATEST} ${DRUNTIME_DIR}-${LATEST} ${PHOBOS_DIR}-${LATEST} ${STABLE_DMD_ROOT} ${DUB_DIR}
 
+RSYNC_FILTER=-f 'P /Usage' -f 'P /.dpl_rewrite*'
```

Comment:
Yes I did.
There is a test folder owned by you, but it doesn't seem to be needed.